README.md

```diff
@@ -82,6 +81,138 @@ kubectl delete namespace $DESIREDNAMESPACE
 
 For more information on running and tearing down k8s environments, follow this [guide](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/docs/running-a-cluster.md).
 
+## Nginx-ingress Custom Configuration
+
+By default, this chart deploys the [nginx-ingress chart](https://github.com/kubernetes/charts/tree/master/stable/nginx-ingress) with the following configuration:
+
+```yaml
+nginx-ingress:
+  config:
+    ssl-redirect: "false"
+  controller:
+    scope:
+      enabled: true
+```
+
+If you want to customize the SSL certificate on the ingress level you can choose one of the options below:
+
+<details>
+<summary>Option 1</summary>
+<p>
+
+If you want your own certificate set on the ELB created through AWS you should create a secret from your cert files:
+
+```bash
+kubectl create secret tls certsecret --key /tmp/tls.key --cert /tmp/tls.crt \
+  --namespace $DESIREDNAMESPACE
+```
+
+Then deploy the infrastructure with following settings:
+
+```bash
+cat <<EOF > infravalues.yaml
+#Persistence options
+persistence:
+  #Enables the creation of a persistent volume
+  enabled: true
+  efs:
+    #Enables EFS ussage
+    enabled: false
+    #DNS address of EFS
+    dns: fs-example.efs.us-east-1.amazonaws.com
+    #Base path to use within the EFS that is mounted as a volume
+    path: "/"
+  #Size allocated to the volume in K8S
+  baseSize: 20Gi
+
+nginx-ingress:
+  controller:
+    config:
+      ssl-redirect: "false"
+    scope:
+      enabled: true
+    publishService:
+      enabled: true
+    extraArgs:
+      default-ssl-certificate: $DESIREDNAMESPACE/certsecret
+EOF
+
+helm install alfresco-incubator/alfresco-infrastructure \
+-f infravalues.yaml \
+--namespace $DESIREDNAMESPACE
+```
+
+</p>
+</details>
+
+<details>
+<summary>Option 2</summary>
+<p>
+
+If you
+
+* created the cluster in AWS using [kops](https://github.com/kubernetes/kops/)
```

Comment:
If you drop the reference to kops here and just say in AWS it will apply to EKS as well.